Lib/test/test_bytes.py

```diff
@@ -14,6 +14,7 @@
 import tempfile
 import textwrap
 import unittest
+from ast import literal_eval as _literal_eval
```

Comment:
Why the underscore?

Reply:
> Why the underscore?

I copied a pattern I saw in other test programs.  It isn't necessary since literal_eval is not used anywhere else in the file.  I will remove it.